client/src/pages/Login/index.tsx

```diff
@@ -34,13 +34,13 @@ function Login() {
   });
 
   return (
-    <div className="loginPage">
+    <div className="signPage">
```

Comment:
not sure why you change classNames

Reply:
to use one style file in two pages: login and signup